project/src/main/puzzle/combo-tracker.gd

```diff
@@ -6,15 +6,11 @@ Keeps track of the player's combo.
 Increments the combo when the player does well, breaks the combo when the player messes up.
 """
 
-signal combo_changed(value)
 signal combo_break_changed(value)
 
 # bonus points which are awarded as the player continues a combo
 const COMBO_SCORE_ARR = [0, 0, 5, 5, 10, 10, 15, 15, 20]
 
-# number of lines the player has cleared without dropping their combo
```

Comment:
Preserve this comment; the comment in PuzzleScore.combo is less precise

Reply:
PuzzleScore.combo description has been updated, ` # The number of lines the player has cleared without dropping their combo`